src/plots/cartesian/axes.js

```diff
@@ -819,6 +823,13 @@ axes.calcTicks = function calcTicks(ax) {
         vals.push(x);
     }
 
+    // If same angle over a full circle, the last tick vals is a duplicate.
+    //
+    // TODO must do something similar for angular date axes.
+    if(ax._id === 'angular' && Math.abs(rng[1] - rng[0]) === 360) {
```

Comment:
I had to hack a few things deep into `axes.js` to get the angular axes looking the way I wanted. I hope @alexcjohnson won't mind.

Reply:
What if you use something else as the period of a linear angular axis? Also do we need to do something more flexible to handle *nearly* duplicate values so we don't get overlaps?